client/src/components/Navbar/ProfileButton.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import './Navbar.css';
+import { useMediaQuery } from 'react-responsive';
+
+function ProfileButton(props) {
```

Comment:
You could destructor the props here: ProfileButton(props) -> ProfileButton({image, name, username})